rskj-core/src/main/java/org/ethereum/core/Transaction.java

```diff
@@ -60,6 +63,9 @@
  */
 public class Transaction {
     public static final int DATAWORD_LENGTH = 32;
+    private static final String TYPED = "7c"; //EIP-2718
+    public static final byte LEGACY_TYPE = 0;
+    protected static final byte AA_TYPE = 1;
```

Comment:
Minor: could be problematic with wallet/ethers tools which may assume type 1 as EIP2930 (or is it 1559).